drs_cli/client.py

```diff
@@ -30,18 +33,27 @@ class DRSClient():
     def __init__(
         self,
         host: str = 'http://0.0.0.0',
-        port: int = '8080',
+        port: int = 80,
         base_path: str = 'ga4gh/drs/v1',
+        drs_uri: Optional[str] = None,
```

Comment:
From the user's perspective, it might be easier to have the first parameter (you could call it 
`uri`) take either the API host _or_ a DRS URI. As the port is usually 80 or 443 (you can set that based on whether the user supplies a host with `http` or `https`, respectively), and the base path is usually the same (as it is defined in the API specs), the user could call the constructor in most cases just with a single positional parameter, passing either host or DRS URI, e.g., `DRSClient("drs://my-drs.org/A3D4SR")` or `DRSClient("https://my-drs.org")`. You can then call your `_check_drs_uri_regex()` and if that fails, call a `_check_http_uri_regex()` and if both fail, you give an error message that says that the passed connection cannot be interpreted and that either a HTTP(S) or DRS URI needs to be passed.